config/mongoid.yml

```diff
@@ -14,7 +14,6 @@ production:
   database: govuk_content_production
   refresh_mode: :sync
   logger: false
-  refresh_interval: 120
```

Comment:
> However, maybe the production config for Mongoid in the repo isn't actually used in production...?

It is indeed used, @floehopper. Thanks for investigating so thoroughly. I'll come back to this later.